internal/partners/prepare.balances.go

```diff
@@ -65,5 +63,5 @@ func computeDefaultBalance(
 	decimals := tokens.Store.Tokens[chainID][vaultAddress].Decimals
 	_, fBalance := helpers.FormatAmount(balance.String(), int(decimals))
 
-	return balance, fBalance, err
+	return bigNumber.FromInt(balance), fBalance, err
```

Comment:
should we prefer using `SetInt()` now since it does a double check on the values?